content/en/docs/examples/virtual-machines/multi-network/index.md

```diff
@@ -30,117 +30,122 @@ bare metal and the clusters.
+            port: 443
+        vm-network:
+
+EOF
     {{< /text >}}
 
-1. For a simple setup, deploy Istio control plane into the cluster
-
-        {{< text bash >}}
-        $ istioctl install \
-            -f manifests/examples/vm/values-istio-meshexpansion.yaml
-        {{< /text >}}
-
-    For further details and customization options, refer to the
-    [installation instructions](/docs/setup/install/istioctl/).
-
-### Setting up the VM
-
-Next, run the following commands on each machine that you want to add to the mesh:
-
-1.  Copy the previously created `cluster.env` and `*.pem` files to the VM. For example:
+2. Install with the virtual-machine features enabled:
 
     {{< text bash >}}
-    $ export GCE_NAME="your-gce-instance"
-    $ gcloud compute scp --project=${MY_PROJECT} --zone=${MY_ZONE} {key.pem,cert-chain.pem,cluster.env,root-cert.pem} ${GCE_NAME}:~
+$ istioctl install -f ./vmintegration.yaml
     {{< /text >}}
 
-1.  Install the Debian package with the Envoy sidecar.
-
-    {{< text bash >}}
-    $ gcloud compute ssh --project=${MY_PROJECT} --zone=${MY_ZONE} "${GCE_NAME}"
-    $ curl -L https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/deb/istio-sidecar.deb > istio-sidecar.deb
-    $ sudo dpkg -i istio-sidecar.deb
-    {{< /text >}}
+#### Specify the network for the VM sidecar
 
-1.  Add the IP address of Istio gateway to `/etc/hosts`. Revisit the [preparing the cluster](#preparing-the-kubernetes-cluster-for-vms) section to learn how to obtain the IP address.
-The following example updates the `/etc/hosts` file with the {{< gloss >}}Istiod{{< /gloss >}} address:
+When following the [Virtual Machine Installation](/docs/setup/install/virtual-machine/) setup guide for creating the `sidecar.env` file we need to tweak the installation by adding the following entry:
 
     {{< text bash >}}
-    $ echo "${GWIP} istiod.istio-system.svc" | sudo tee -a /etc/hosts
+$ echo "ISTIO_META_NETWORK=vm-network" >> sidecar.env
     {{< /text >}}
 
-   A better options is to configure the DNS resolver of the VM to resolve the address, using a split-DNS server. Using
-   /etc/hosts is an easy to use example. It is also possible to use a real DNS and certificate for Istiod, this is beyond
-   the scope of this document.
+#### Create Gateway for application traffic
 
-1.  Install `root-cert.pem`, `key.pem` and `cert-chain.pem` under `/etc/certs/`.
+The last step is to create a `Gateway` resource that allows application traffic from the VMs to route correctly. 
 
-    {{< text bash >}}
-    $ sudo mkdir -p /etc/certs
-    $ sudo cp {root-cert.pem,cert-chain.pem,key.pem} /etc/certs
-    {{< /text >}}
+    {{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: cluster-aware-gateway
+  namespace: istio-system
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 443
+      name: tls
+      protocol: TLS
+    tls:
+      mode: AUTO_PASSTHROUGH
```

Comment:
@nmittler I forgot the latest on this discussion. Does this align with the future plans for MC docs?

Reply:
Let me know what we should do here.

Reply:
Yes, this is what we're proposing for Istio 1.7: https://github.com/istio/istio/blob/master/samples/cross-network-gateway/cross-network-gateway.yaml

It's the best solution we have for now, unfortunately.